docs/docs-components/AccessibilitySection.js

```diff
@@ -13,7 +13,7 @@ type Props = {|
   name: string,
 |};
 
-function AccessibilityMainSection({
+export default function AccessibilitySection({
```

Comment:
Not strictly in scope for these changes, but I noticed the default export name didn't match the file name